packages/camera/camera_android_camerax/lib/src/system_services.dart

```diff
@@ -61,6 +61,19 @@ class SystemServices {
 
     api.stopListeningForDeviceOrientationChange();
   }
+
+  /// Returns a file path which was used to create a temporary file.
+  ///
+  /// The file and path constraints are determined by the implementation of
+  /// File.createTempFile(prefix, suffix, cacheDir), on the android side, where
+  /// where cacheDir is the cache directory identified by the current application
+  /// context using context.getCacheDir().
+  static Future<String> getTempFilePath(String prefix, String suffix,
```

Comment:
I think from the tests I read prefix is actually the file name and suffix is the extension. That came as a bit of a suprise from reading the documentation I would have expected `prefix<something the caller does not control>suffix` for the file name. 

For example it means as a caller I think this means I need to ensure that I pass something different each time  or risk overwriting the previous file.

Reply:
I'll clarify the documentation before merging, the way it actually works is that the file ends up being
`prefix<something the caller does not control.suffix`, i.e. 'suffix' is the file extension, and prefix is _part_ of the file name, but more gets added by createTempFile (a random sequence of numbers).

So `getTempFile('MOV', '.temp')`, is something like `<cachedir>/MOV1231242244.temp`.

FWIW I chose prefix and suffix because those are the variable names used by the native methods implementation/documentation.

Reply:
OK in that case it means the documentation *is correct* but that is not what the tests are testing for.  I think using the same names as the native code is a reasonable choice. 

Specifically in packages/camera/camera_android_camerax/test/system_services_test.dart and in packages/camera/camera_android_camerax/android/src/test/java/io/flutter/plugins/camerax/SystemServicesTest.java you are testing for the result to be equal to`testPath + testPrefix + testSuffix` (or the file for prefix and suffix). Instead of testing that the output had a path of `testPath` a file prefix of `testPrefix` and an extension of `testSuffix`. Which is fine it is just confusing when trying to understand what to expect from the tests.